### PR TITLE
My portfolio

### PR DESCRIPTION
**Pull request by Kadunapikin**
my html-css-portfolio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,161 @@
...
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "started"><b>Getting Started</b></h2>
+
+---
+To assess this webpage, here are simple steps to follow:
+
+1. Open your browser
+2. In the search or input box, type in the portfolio web address <https://github.com/Kadunapikin/portfolio/> and you will be directed to the home page.
```
Comment (zizoh) [inline README.md]: This link doesn't open a valid webpage. Please check it.

Comment (Kadunapikin) [reply README.md]: Link has been updated now sir

File: README.md [hunk 2]
```diff
@@ -1,2 +1,161 @@
...
+
+---
+<h2 id = "end"><b>Acknowledgements</b></h2>
+
+---
+
+- [Zizoh Anto](https://github.com/zizoh)
+- [Anto Akoke Victor](https://github.com/antoHero)
+- [Daniel Samuel](https://github.com/DanSam5K)
+- [Hope Jonah](https://www.)
```
Comment (zizoh) [inline README.md]: I believe this is Hope's website: https://www.hopejonah.live

File: portfolio-assets/Landing page.JPG
Comment (zizoh) [inline portfolio-assets/Landing page.JPG:1]: Please use Google's JavaScript style for this. "File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation. Follow the convention that your project uses": https://google.github.io/styleguide/jsguide.html#file-name.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+function showNavItems() {
+  for (let i = 0; i < navItems.length; i += 1) {
+    navItems[i].style.opacity = '1';
+    navItems[i].style.transform = 'translateY(0)';
+  }
+}
+
+window.addEventListener('load', showNavItems);
+
+function showNavItem() {
```
Comment (zizoh) [inline script.js]: Please what does this function do?

File: README.md (hunk 2 above)
Comment (Kadunapikin) [reply README.md]: Ok!

File: portfolio-assets/Landing page.JPG
Comment (Kadunapikin) [reply portfolio-assets/Landing page.JPG:1]: ok!

File: script.js (hunk 3 above)
Comment (Kadunapikin) [reply script.js]: It was for the mobile responsiveness but i've updated the function now to a different one

**Pull request merged**